### PR TITLE
Update/clean documentation

### PR DESCRIPTION
**Pull request by jmelot**
* Remove outdated references to simhash
* Consolidate documentation files
* Remove old/unused CNKI query sequence file
* Generally gussy up

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,29 @@
 # Article Linking
 ![Python application](https://github.com/georgetown-cset/article-linking/workflows/Python%20application/badge.svg)
 
-This repository contains a description and supporting code for CSET's current method of
-cross-dataset article linking. Note that we use "article" very loosely, although in a way that to our knowledge
-is fairly consistent across corpora. Books, for example, are included.
+At CSET, we aim to produce a more comprehensive set of scholarly literature by ingesting multiple sources and then
+deduplicating articles. This repository contains CSET's current method of cross-dataset article linking. Note that we
+use "article" very loosely, although in a way that to our knowledge is fairly consistent across corpora. Books, for
```
Comment (billsponsor) [inline README.md:6]: Is "consistent across corpora" meant to say that "articles" are all referring to the same type of thing, regardless of the source dataset? Or meant to say that our definition of "article" matches similar definitions across other aggregated pubs datasets?

File: README.md [hunk 2]
```diff
@@ -1,13 +1,29 @@
...
+article linkage, see the [ETO documentation](https://eto.tech/dataset-docs/mac/).
+
+## Matching articles
+
+To match articles, we need to extract the data that we want to use in matching and put it in a consistent format. The
+SQL queries specified in the `sequences/generate_{dataset}_data.tsv` files are run in the order they appear in those
+files. For OpenAlex we exclude documents with a `type` of Dataset, Peer Review, or Grant. Additionally, we take every
+combination of the Web of Science titles, abstracts, and pubyear so that a match on any of these combinations will
+result in a match on the shared WOS id. Finally, for Semantic Scholar, we exclude any documents that have a non-null
+publication type that is one of Dataset, Editorial, LettersAndComments, News, or Review.
```
Comment (billsponsor) [inline README.md:21]: Are all entries from arXiv & PWC taken without a pub type restriction? Reads a bit weird to specify the steps for OA, WOS, and SemScholar, but not arXiv & PWC.

File: README.md [hunk 3]
```diff
@@ -20,26 +36,34 @@ as well as a match on one additional field above, or on
...
-- `gcp_cset_links_v2.all_metadata_with_cld2_lid` - provides CLD2 LID for the titles and abstracts of each
-current version of each article's metadata. You can also use this table to get the metadata used in the
-match for each version of the raw articles. Note that the `id` column is _not_ unique as some corpora like WOS
-have multiple versions of the metadata for different languages.
+The DAG generates two tables of analytic significance:
 
-- `gcp_cset_links_v2.article_merged_metadata` - This maps the CSET `merged_id` to a set of merged metadata.
-The merging method takes the maximum value of each metadata field across each matched article, which may not
-be suitable for your purposes.
+* `staging_literature.all_metadata_with_cld2_lid` - captures metadata for all unmerged articles in a
```
Comment (billsponsor) [inline README.md:67]: Potentially highlight what the metadata here looks like - for example, how to select titles/abstracts that are linked but might be different (my recollection is we take the longest available one), keeping the original language title/abstract if available, etc. External readers won't have the ability to go look at what's in the table.

File: README.md (hunk 1 above)
Comment (jmelot) [reply README.md:6]: Yeah, the first one - I'll make that clearer

File: README.md (hunk 2 above)
Comment (jmelot) [reply README.md:21]: Yes, they're taken without restriction

File: README.md (hunk 3 above)
Comment (jmelot) [reply README.md:67]: Metadata selection happens downstream in https://github.com/georgetown-cset/cset_article_schema so I think this documentation belongs there, but I'll add a link to that repo which should help clarify that this work happens elsewhere.

**Comment by jmelot** [comment]
Thanks @billsponsor !

**Pull request merged**